NitroxModel/MultiplayerSession/MultiplayerSessionReservationState.cs

```diff
@@ -22,6 +22,9 @@ public enum MultiplayerSessionReservationState
 
         [Description("The server is using hardcore gamemode, player is dead.")]
         HARDCORE_PLAYER_DEAD = 1 << 4,
+
+        [Description("Another user is currently joining the server.")]
```

Comment:
We can't join the server is someone is already joining ? This player shouldn't be just queued ?

Reply:
This reservation state lets the client know that they have been enqueued in the join queue. They are then dequeued by another MultiplayerSessionReservation that lets them know if they have been reserved.